src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalAnalysisResult.js

```diff
@@ -488,7 +491,8 @@ export default compose(
         nextControlPanelExpanded === controlPanelExpanded &&
         nextLoading === loading &&
         isEqual(populateSurvivalData, nextPopulateSurvivalData) &&
-        nextSurvivalDataLoading === survivalDataLoading
```

Comment:
added diff check for `currentAnalysis` here, because `overallSurvival` is no longer listening to changes to `currentAnalysis` - without this, the page won't update when you make custom bins or switch to a different case set.